views/search/logs.ejs

```diff
@@ -30,7 +30,7 @@
                 /u/{{result.user}}
               </h4>
               <p class="list-group-iitem-text">
-                {{result.content}}. {{result.createdAt | date:'yyyy-MM-dd HH:mm:ss'}} UTC
+                {{result.content}}. {{result.createdAt | date:'yyyy-MM-dd HH:mm:ss UTCZ'}}
```

Comment:
We may as well fix this, since at the moment it's blatantly inaccurate if the browser timezone is not UTC.

With this fix it looks like [this](https://i.gyazo.com/7564e3d7c2577448ade75d314691624f.png).